pkg/model/types.go

```diff
@@ -123,6 +135,16 @@ func servicePortFromAttr(attributes map[string]string) (port Port, err error) {
 	return port, err
 }
 
+func clusterIdFromAttr(attributes map[string]string) (clusterId string, err error) {
```

Comment:
What is the benefit of creating wrapper functions for these parse actions instead of direct calls to `removeStringAttr()` like getting `endpoint.IP` does?